src/botPage/view/Dialogs/Limits.js

```diff
@@ -3,33 +3,51 @@ import PropTypes from 'prop-types';
 import { translate } from '../../../common/i18n';
 import * as style from '../style';
 import Dialog from './Dialog';
+import { restrictInputCharacter } from '../shared';
 
 class LimitsContent extends PureComponent {
     constructor() {
         super();
         this.state = {
-            error: null,
+            error    : '',
+            maxLoss  : '',
+            maxTrades: '',
         };
     }
     submit() {
-        const maxLoss = +this.maxLossDiv.value;
-        const maxTrades = +this.maxTradesDiv.value;
-        if (maxLoss > 0 && maxTrades > 0) {
-            if (maxTrades <= 100) {
-                this.props.onSave({
-                    maxLoss,
-                    maxTrades,
-                });
-            } else {
-                this.setState({
-                    error: translate('Maximum allowed number of trades for each session is 100.'),
-                });
-            }
-        } else {
-            this.setState({
-                error: translate('Both number of trades and loss amount have to be positive values.'),
-            });
+        let { maxLoss } = this.state;
```

Comment:
If you want to keep this as both string and number, better to create a variable like `maxLossStr`. Use `let` only when the variable is meant to be changed (and avoid that whenever possible).